README.md

```diff
@@ -1 +1,62 @@
-# positioning-and-floating-elements
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Positioning and Floating Elements project: New York Times article page
+
+> This is a New York Times article page clone using HTML and CSS3.
+
+## Screenshots of original website
+
+![screenshot](<./ny-time-screenshot(2).png>)
+
+## Screenshots of our clone website
+
+![screenshot](<./ny-time-screenshot(3).png>)
+
+The project is built using Html and Css with an extensive use of the grid, flex and float css properties.
+
+## Built With
+
+- HTML
+- CSS3
+- Font awesome icons
+
+## Live Demo
+
+[Live Demo Link](https://rawcdn.githack.com/RICKCOYL/positioning-and-floating-elements/336ee37158eebc225224d52a850578a766ba2986/index.html)
+
+## To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+Clone the project locally.
+
+### Linters
+
+1. run `npm install`.
+2. run `npm run test` to check the Html and Css files.
+
+### Deployment
+
+Install and run a live server plugin on you IDE/Text editor and run it from the root directory.
+
+## Authors
+
+👤 **SHONIBARE COMFORT**
+
+- Github: [@githubhandle](https://github.com/Adewunmi97)
+- Twitter: [@twitterhandle](https://twitter.com/ShonibareC)
+- Linkedin: [linkedin](https://www.linkedin.com/in/ceamatu-cristian-viorel-7a5469136/)
+
+👤 **RICK OBURU**
+
+- Github: [@githubhandle](https://github.com/RICKCOYL)
+- Twitter: [@twitterhandle](https://twitter.com/rickoburu)
+- Linkedin: [linkedin](https://linkedin.com/in/rick-oburu-8627591a4)
```

Comment:
- [ ] Please change where it says "githubhandle / twitterhandle / linkedin" for your own usernames in those social networks